p/scripts/main.js

```diff
@@ -1001,12 +1001,16 @@ function init_shortcuts() {
 			if (context.auto_mark_site) {
 				mark_read(document.querySelector('.flux.current'), true, false);
 			}
-			const newWindow = window.open();
-			if (newWindow) {
-				newWindow.opener = null;
-				newWindow.location = document.querySelector('.flux.current a.go_website').href;
+
+			const link_go_website = document.querySelector('.flux.current a.go_website');
+			if (link_go_website) {
+				const newWindow = window.open();
+				if (newWindow) {
```

Comment:
I'd personally be more inclined to write it more like this as I think that's slightly more legible but I suppose ymmv. :-)
```suggestion
			const link_go_website = document.querySelector('.flux.current a.go_website');
			const newWindow = window.open();
			if (link_go_website && newWindow) {
```